app/Http/Controllers/CourseController.php

```diff
@@ -38,9 +69,29 @@ public function create()
         return view('supervisor.manage-course.create-course');
     }
 
-    public function store(Request $request)
+    public function store(CourseRequest $request)
     {
+        $course = Course::create([
+            'title' => $request->title,
+            'image' => $request->image->getClientOriginalName(),
+            'description' => $request->content_description,
+            'created_at' => now()->format(config('view.format_date.datetime')),
+            'status' => config('number.subject.active'),
+        ]);
+
+        for ($item = 0; $item < count($request->titleSubject); $item++) {
+            Subject::create([
+                'title' => $request->titleSubject[$item],
+                'image' => '',
```

Comment:
sao ko dùng foreach mà lại dùng for?

Reply:
đoạn này em dùng cả phần tử cùng stt của mảng khác nữa nên e dùng vòng for ạ

Reply:
c có thấy em dùng chỗ nào ddaau?

Reply:
có 2 mảng là timeSubject cả titleSubject ạ